### PR TITLE
chore(driving-licence-book): Refactor configModule for driving licence and driving licence book

### PR DESCRIPTION
**Pull request by rafnarnason**
configModule upgrade and improved documentation

Attach a link to issue if relevant

upgrading clients to use configmodule library

Specify what you're trying to achieve

Recommended by Core and better readmes are always better

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/clients/driving-license-book/src/lib/drivingLicenseBookClient.module.ts [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+import { exportedApis } from './apis'
 import { DrivingLicenseBookClientApiFactory } from './drivingLicenseBookClient.service'
 
+// what check bare minimum
+
 @Module({
   providers: [DrivingLicenseBookClientApiFactory],
   exports: [DrivingLicenseBookClientApiFactory],
 })
+// @Module({
```
Comment (alexdiljar) [inline libs/clients/driving-license-book/src/lib/drivingLicenseBookClient.module.ts:12]: remove comment?

Comment (rafnarnason) [reply libs/clients/driving-license-book/src/lib/drivingLicenseBookClient.module.ts]: totally

**Review by darrikonn — APPROVED** [review]

**Review by obmagnusson — APPROVED** [review]
LGTM. Although you might want to add the `libs/application/template-api-modules/src/lib/modules/templates/driving-school-confirmation/` folder to your team in the codeowners file.

File: libs/clients/driving-license-book/README.md [hunk 2]
```diff
@@ -23,8 +23,54 @@ yarn nx run clients-driving-license-book:update-openapi-document
 yarn nx run clients-driving-license-book:schemas/external-openapi-generator
 ```
 
-### Import into other NestJS modules
+### Import into other NestJS modules \***\*\*\*\*\***\*\*\*\*\***\*\*\*\*\***\*\***\*\*\*\*\***\*\*\*\*\***\*\*\*\*\***
```
[the remark this answers is not in the document]
Comment (rafnarnason) [reply libs/clients/driving-license-book/README.md]: just looks stunning ;) .... scratch that ... gone.

**Pull request merged**